client/src/components/Tool/ToolSuccess.vue

```diff
@@ -31,7 +31,7 @@ onMounted(() => {
 
 <template>
     <section>
-        <div v-if="jobResponse.produces_entry_points">
+        <div v-if="jobResponse?.produces_entry_points">
             <ToolEntryPoints v-for="job in jobResponse.jobs" :key="job.id" :job-id="job.id" />
         </div>
         <ToolSuccessMessage :job-response="jobResponse" :tool-name="toolName" />
```

Comment:
This might also need a v-if or a loading span maybe ?

Reply:
By this do you mean the `ToolSuccessMessage`?

Reply:
Right, since `jobResponse` appears to be undefined sometimes (https://sentry.galaxyproject.org/share/issue/a383eb1f6bf34eeeb17b7717b9f6e510/)?

Reply:
Which is the case if a user navigates directly to https://usegalaxy.org/jobs/submission/success for instance, and https://github.com/galaxyproject/galaxy/blob/dev/client/src/components/Tool/ToolSuccess.vue#L24-L29 kicks in ... which seems at least a little unconventional.

Reply:
It would be so cool if we were able to check for a job response directly in the router and immediately redirect from there, but unfortunately we can't access the `jobStore` in the router...